### ISSUE TITLE
No console output

### ISSUE THREAD

**Issue by sierdzio** [probonopd/linuxdeployqt#178]
Hi, a very weird issue started happening for me today, on 2 separate PCs. linuxdeployqt suddenly stopped printing **any** output, even if I only call
```./linuxdeployqt.AppImage -help```

Even more weirdly, this happens both to newest release, and an older build I have (from beginning of October), which used to work flawlessly.

Is this an issue with linuxdeployqt somehow, or maybe rather something on my side? I'm really baffled as this started happening simultaneously on 2 PCs.

**Comment by sierdzio** [comment]
Additional info: after exporting ```QT_LOGGING_RULES="*.debug=true"``` console output is visible again.

So, I'm not sure if it is linuxdeployqt issue (I've run it with -verbose=1 just before the problem started happening, maybe it overwrites some config file?) or something got messed up on my side. Feel free to close this ticket or investigate further.

**Comment by sierdzio** [comment]
No, my env contains only:
```
QT_QUICK_CONTROLS_STYLE=org.kde.desktop
QT_ACCESSIBILITY=1
QT_AUTO_SCREEN_SCALE_FACTOR=0
```

I've noticed that the logging rule has been set up in file /etc/xdg/QtProject/qtlogging.ini with contents:
```
[Rules]
*.debug=false
```
I have no idea what created this file (I install Qt as local user only, and I use linuxdeployqt also only as local user, not root), but removing it also fixes the issue.

Bottom line: I don't know what caused this, and only limited evidence points to linuxdeployqt (the fact that it happened on 2 machines where I run linuxdeployqt). So the bug report is probably invalid.

**Comment by probonopd** [comment]
I don't think that linuxdeployqt has anything to do with creating `/etc/xdg/QtProject/qtlogging.ini`, at least I am not aware of ever having seen that file.

**Comment by probonopd** [comment]
I can confirm that Ubuntu 17.10 contains a file `/etc/xdg/QtProject/qtlogging.ini` by default that, when existing, prevents `linuxdeployqt` from producing any output.
